files/__addonLocation__/package.json

```diff
@@ -10,18 +10,25 @@
   "author": "",
   "files": [
     "addon-main.cjs",
-    "dist"
+    "dist",
+    "declarations"
   ],
-  "scripts": {
-    "build": "rollup --config",
+  "scripts": {<% if (typescript) { %>
+    "build": "concurrently 'npm:build:*'",
+    "build:js": "rollup --config",
+    "build:types": "glint --build", <% } else { %>
+    "build": "rollup --config", <% } %>
```

Comment:
What do you think of keeping concurrently for both TS and JS builds? that way we can just add the `build:types` in case of typescript

Just an idea

Reply:
idk, if we keep each convenience thing for TS in the JS version of the addon, we'd probably have a lot less conditionals all over the place.

Since this is blocking other work, how do you feel about discussing in another pr?

Reply:
don't think of any of my comments as a blocker 👍 they are only minor suggestions that are more helpful as DX for us than anything else